external-plugins/github-jira-proxy/Makefile

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
I think we can consume from your own GitHub repository for a start. Other than that, this folder is reserved for proper external prow plugins, which the code here doesn't match (yet). Having said that, you can either build from your public repo in the `images` Dockerfile, or move your code to the `robots` folder.